gnovm/pkg/gnolang/values_conversions_test.go

```diff
@@ -0,0 +1,25 @@
+package gnolang
+
+import (
+	"math"
+	"testing"
+
+	"github.com/cockroachdb/apd"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConvertUntypedBigdecToFloat(t *testing.T) {
```

Comment:
You can make probably make this test parallel `t.Parallel()`

Reply:
Having some flashbacks... :stuck_out_tongue_winking_eye:  : https://github.com/gnolang/gno/pull/978#discussion_r1276281061